.github/ISSUE_TEMPLATE/bug_report_template.md

```diff
@@ -12,43 +12,34 @@ assignees: ''
 Thank you for submitting an issue. Please refer to our [issue policy](https://www.github.com/voxel51/fiftyone/blob/develop/ISSUE_POLICY.md) for information on what types of issues we address.
 
 1. Please fill in this template to ensure a timely and thorough response
-2. Place an "x" between the brackets next to an option if it applies. For example:
+2. Remove the section instructions but leave the section header
+3. Place an "x" between the brackets next to an option if it applies. For example:
     -   [x] Selected option
-3. **Please delete everything above this line before submitting the issue**
+4. **Please delete everything above this line before submitting the issue**
 
-### System information
+### Describe the problem
 
--   **OS Platform and Distribution** (e.g., Linux Ubuntu 16.04):
--   **Python version** (`python --version`):
--   **FiftyOne version** (`fiftyone --version`):
--   **FiftyOne installed from** (pip or source):
+Describe the problem clearly here. Include descriptions of the expected behavior and the actual behavior.
 
-### Commands to reproduce
```

Comment:
It's not clear to me that the distinction between "Commands to reproduce" and "Code to reproduce issue" is all that meaningful. So let's just make it simpler by having description above, space for reproducing steps (code, screenshots/videos, instructions, etc.), and additional info section down below if needed.